submissions/chimoney-redeem-airtime/src/components/Redeem.js

```diff
@@ -0,0 +1,199 @@
+
+            //check if the first digit is "+" and it is a valid phone number
+            if (firstDigit === '+' && CheckPhone) {
+
+                //remove unneccessary symbols from phone number and
+                // concatenate it with '+' to form a perfect phone number
+                Phone = '+' + Phone.replace(/\D/g, '');
+
+                let config = {
+                    method: 'POST',
+                    url: 'https://api.chimoney.io/v0.2/redeem/airtime',
+                    headers: {'X-API-Key': API_KEY },
+                    data: formDetails
+                  };
+                  
+                  axios(config)
+                    .then(function (response) {
+                        // console.log(JSON.stringify(response.data));
+                        setLoading(false);
+                        setformDetails({});
+                        setTicketExist(false)
+                        swal({
+                            title: 'Airtime Redeemed Successfully!',
+                            showClass: {
+                              popup: 'animate__animated animate__fadeInDown'
+                            },
+                            hideClass: {
+                              popup: 'animate__animated animate__fadeOutUp'
+                            }
+                          })
+                    })
+                      .catch(function (error) {
+                        setLoading(false);
+                        seterrorMsg('confirm ticket has not been used');
+                        setTicketExist(false)
+                        console.log(error)
+                    });
+            } else {
+                setLoading(false);
+                seterrorMsg('make sure your phone number follows the correct format')
+            }
+        }
+    }
+    const clearErrorMsg = () => {
+        seterrorMsg('')
+    }
+
+  return (
+      <FormContainer>
+         <h1>Convert Chimoney to Airtime</h1>
+          <form onMouseDown={clearErrorMsg} onKeyDown={clearErrorMsg}>
+              {errorMsg ? <p className='error'>{errorMsg}</p> : ''}
+            
+              <div>
+              <p>Ticket ID</p>
+                  <input type="text" value={formDetails.chiRef} name="chiRef" onChange={handleChange} id="country" placeholder='enter ticket  id' disabled={ticketExist} />
+              </div>
+              <div>
+              <p>Phone number (+2349023..)</p>
+                <input type="tel" name="phoneNumber" onChange={handleChange} id="phone-number" placeholder='enter phone number' />
+              </div>
+              <div>
+                  <p>Country</p>
+                  <select name="countryToSend" value={formDetails.countryToSend} id="country" onChange={handleChange} >
+                        <option value=''>Select Your Country</option>
+                        {countries.map(({name, value}) => (       

```

Comment:
So there is an endpoint that returns [all countries that Chimoney supports for Airtime](https://chimoney.readme.io/reference/get_v0-2-info-airtime-countries). Could you use that so you do need to hardcode the countries

Because if a user picks a country from this that Chimoney does not support, it will fail